ml-testing-toolkit-cli/values.yaml

```
@@ -93,6 +95,18 @@ postInstallHook:
   weight: -5
   deletePolicy: before-hook-creation
 
+job:
+  enabled: false
+  ttlSecondsAfterFinished: 50
```

I would comment this out
```suggestion
  ## Set the TTL for Job Cleanup - ref: https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/
  # ttlSecondsAfterFinished: 50
```